packages/audio/pipewire/package.mk

```diff
@@ -86,6 +86,13 @@ post_install() {
   # note that the pipewire user is added to the audio and video groups in systemd/package.mk
   # todo: maybe there is a better way to add users to groups in the future?
 
+  # config dir in /storage/.config/
+  mkdir -p $INSTALL/usr/config/pipewire
+
+  # symlink for config
+  mkdir -p $INSTALL/etc
+  ln -s /storage/.config/pipewire $INSTALL/etc/pipewire
+
```

Comment:
We normally avoid creating symlinks in SYSTEM that point to /storage. It would be preferrable to change the configuration location to /storage/.config/pipewire via compile options if pipewire supports that? or to patch pipewire sources to relocate a hardcoded location, or better: contribute a patch to add the compile option and upstream it, then patch pipewire sources with that patch and use build-time config. Make sense?